<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix2/wind/parrot.svg</normaloff>
    <normalon>:/new/prefix2/wind/parrot.svg</normalon>:/new/prefix2/wind/parrot.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut_2"/>
    <addaction name="actionPaste_2"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionRead_only"/>
    <addaction name="actionOver_write"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuHighlight">
     <property name="title">
      <string>Highlight</string>
     </property>
     <addaction name="actionHTML"/>
     <addaction name="actionC"/>
     <addaction name="actionPython"/>
     <addaction name="actionJava_script"/>
    </widget>
    <widget class="QMenu" name="menuConvert">
     <property name="title">
      <string>Convert </string>
     </property>
     <addaction name="actionTo_lower"/>
     <addaction name="actionTo_Upper"/>
     <addaction name="actionTo_HTML_escape"/>
    </widget>
    <addaction name="menuHighlight"/>
    <addaction name="menuConvert"/>
    <addaction name="actionInsert_comments"/>
    <addaction name="actionInsert_date"/>
    <addaction name="actionWord_Count"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_HTML"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_to_HTML"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionFollow_me_on_twitter"/>
   </widget>
   <widget class="QMenu" name="menuStyles">
    <property name="title">
     <string>Search</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExport"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuRun"/>
   <addaction name="menuStyles"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionPrint_Preview"/>
   <addaction name="separator"/>
   <addaction name="actionRun_HTML"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normalon>:/new/prefix2/wind/document-new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normalon>:/new/prefix2/wind/document-open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normalon>:/new/prefix2/wind/document-save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>:/new/prefix2/wind/document-save-as.png</normaloff>:/new/prefix2/wind/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="document-print">
     <normalon>:/new/prefix2/wind/document-print.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="document-close">
     <normalon>:/new/prefix2/wind/edit-delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-undo">
     <normalon>:/new/prefix2/wind/edit-undo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-redo">
     <normalon>:/new/prefix2/wind/edit-undo-rtl.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-copy">
     <normalon>:/new/prefix2/wind/edit-copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut_2">
   <property name="icon">
    <iconset theme="edit-cut">
     <normalon>:/new/prefix2/wind/edit-cut.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste_2">
   <property name="icon">
    <iconset theme="edit-paste">
     <normalon>:/new/prefix2/wind/edit-paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset theme="zoom-in">
     <normalon>:/new/prefix2/wind/zoom-in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset theme="zoom-out">
     <normalon>:/new/prefix2/wind/zoom-out.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionRead_only">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Read only</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionOver_write">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Over write</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normalon>:/new/prefix2/wind/edit-select-all-symbolic.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/text-html.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionC">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/text-x-c++src.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionTo_lower">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/view-sort-descending.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>To lower</string>
   </property>
  </action>
  <action name="actionTo_Upper">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/view-sort-ascending.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>To Upper</string>
   </property>
  </action>
  <action name="actionTo_HTML_escape">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/interface_subroutine_gen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>To HTML escape</string>
   </property>
  </action>
  <action name="actionInsert_comments">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/comment_block.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Insert comments</string>
   </property>
  </action>
  <action name="actionInsert_date">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/folder-temp.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Insert Time</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/text-x-python.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionRun_HTML">
   <property name="icon">
    <iconset theme="html">
     <normalon>:/new/prefix2/wind/folder-copycom.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run HTML</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset theme="document-print-preview">
     <normalon>:/new/prefix2/wind/document-print-preview.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionWord_Count">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/logo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Word Count</string>
   </property>
  </action>
  <action name="actionExport_to_HTML">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/text-html-symbolic.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Export to HTML</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix2/wind/help-about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>:/new/prefix2/wind/zoom-out.png</normaloff>:/new/prefix2/wind/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAdd_new">
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionSearch_with_Google">
   <property name="text">
    <string>Search with Google</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionJava_script">
   <property name="text">
    <string>Java script</string>
   </property>
  </action>
  <action name="actionFollow_me_on_twitter">
   <property name="text">
    <string>Follow me on twitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
